app/controllers/hiring_staff/sign_in/email/sessions_controller.rb

```diff
@@ -54,6 +56,10 @@ def update_session_without_urn(options)
     Rails.logger.warn("Hiring staff signed in via fallback authentication: #{options[:oid]}")
```

Comment:
While we're adding UIDs to this controller, let's rename `update_session_without_urn` to e.g. `update_session_without_urn_and_uid` / `without_org_id`